subsys/usb/usb_c/usbc_pe_common.c

```diff
@@ -345,6 +345,9 @@ void pe_message_received(const struct device *dev)
 	struct policy_engine *pe = data->pe;
 
 	atomic_set_bit(pe->flags, PE_FLAGS_MSG_RECEIVED);
+
+	/* This call prevents the USB-C thread from sleeping and decreases the response time */
```

Comment:
Can you add a separate comment to justify each call to `usbc_prevent_sleep_once()`?  A description of the specific behavior that is fixed or spec reference would be helpful.